lib/std/os/uefi.zig

```diff
@@ -1,45 +1,90 @@
+/// A protocol is an interface identified by a GUID.
 pub const protocols = @import("uefi/protocols.zig");
+/// Status codes returned by EFI interfaces
 pub const status = @import("uefi/status.zig");
 pub const tables = @import("uefi/tables.zig");
 
+const fmt = @import("std").fmt;
+
 const builtin = @import("builtin");
 pub const is_the_target = builtin.os == .uefi;
 
+/// The EFI image's handle that is passed to its entry point.
 pub var handle: Handle = undefined;
+/// A pointer to the EFI System Table that is passed to the EFI image's entry point.
 pub var system_table: *tables.SystemTable = undefined;
 
+/// A handle to an event structure.
 pub const Event = *@OpaqueType();
-// GUIDs must be align(8)
+
+/// GUIDs must be align(8)
```

Comment:
@andrewrk can you put `align(8)` on the first member to ensure this?

Reply:
Yes, for extern structs.